docs/dictionary/command/create-card.lcdoc

```diff
@@ -24,10 +24,16 @@ create card "Main"
 Example:
 create card (field "ouput")
 
+Example:
+create card "Main" in stack "Stack"
+
 Parameters:
 cardName:
 The name property of the new card. If you don't specify a cardName, the
 new card's name is empty.
+stackName:
+The stack in which the new card is placed. If you don't specify
+a stackName, it is placed in defaultStack.
```

Comment:
This is probably better as `in the defaultStack`. Also, if you put `<defaultStack>` that creates a link to the defaultStack entry in the docs.